providers/linenotify/README.md

```diff
@@ -0,0 +1,13 @@
+# Novu Linenotify Provider
```

Comment:
Please rename the folder of the provider to `line-notify` instead of `linenotify`.